### PR TITLE
Tatu kassim

### PR DESCRIPTION
**Pull request by Tatu360**

### CODE REVIEW

File: functions.py [hunk 1]
```diff
@@ -2,3 +2,35 @@
 Write a simple script that demonstrate your understanding of functions in python 
 '''
 
+def printme( str):
+    "this print a passed string into this function"
+    print (str)
+    return
+
+    printme("I am first call to user defined function")
```
Comment (younggeeks) [inline functions.py:10]: Are these going to run? 
It looks to me like they are part of printme(str) function

File: operators.py [hunk 2]
```diff
@@ -2,3 +2,12 @@
 Write a simple script that demonstrate your understanding of Operators,operands and operator precedence
 '''
```
Comment (younggeeks) [inline operators.py:3]: Looks good but next time you'll need to add more examples of operators

**Comment by Tatu360** [comment]
Okay i will do my best next time

File: functions.py (hunk 1 above)
Comment (Tatu360) [reply functions.py:10]: they are running

**Pull request merged**